server/moodle/mod/livequiz/view.php

```diff
@@ -1,9 +1,9 @@
 <?php
-// This file is part of Moodle - http://moodle.org/
+// This file is part of Moodle - http://moodle.org/.
 //
-// Moodle is free software: you can redistribute it and/or modify
-// it under the terms of the GNU General Public License as published by
-// the Free Software Foundation, either version 3 of the License, or
+// Moodle is free software: you can redistribute it and/or modify.
+// it under the terms of the GNU General Public License as published by.
+// the Free Software Foundation, either version 3 of the License, or.
```

Comment:
Codesniffer?

Reply:
codesniffer requres docblock for every file, class and function so yeah codesniffer
... the "." was a mistake i missed though i'll get them removed (from the boilerplate stuff) they're requred for inline comments tough